src/custom-elements/src/components/drawer/drawer.tsx

```diff
@@ -192,24 +192,15 @@ export class Drawer {
   }
 
   connectedCallback() {
-    const observerTarget = this.el;
-    const mutationObserverConfig = {
-      attributes: true,
-      attributeOldValue: true,
-      attributeFilter: ['title'],
-    };
-
-    if (!this.mutationObserver) {
-      const subscriberCallback = (mutations) => {
-        mutations.forEach((mutation) => {
-          this.drawerTitle = mutation.target.title;
-        });
-      };
-
-      this.mutationObserver = new MutationObserver(subscriberCallback);
-    }
-
-    this.mutationObserver.observe(observerTarget, mutationObserverConfig);
+    addMutationObserver.call(this, () => {
+        this.drawerTitle = this.el.title;
+      },
+      { attributes: true, 
+        attributeOldValue: true, 
+        attributeFilter: ['title'], 
+        childList: true 
+      }
+    );
```

Comment:
Same as in https://github.com/CenturyLink/Chi/pull/1069/files#diff-3a8fc2545050e66041c7e2318d83b281fdc9a73a512bd77a6baca65dfb7bb801R75

Reply:
https://github.com/CenturyLink/Chi/pull/1069#discussion_r1514264287